okreplay-core/src/main/java/okreplay/OkReplayInterceptor.kt

```diff
@@ -113,7 +115,7 @@ class OkReplayInterceptor : Interceptor {
   }
 
   private fun isHostIgnored(request: okhttp3.Request): Boolean {
-    return configuration!!.ignoreHosts.contains(request.url().host())
+    return configuration!!.ignoreHosts.contains(request.url.host)
```

Comment:
This looks like an unsafe access in non-demo code. Could it be changed to a safe access instead?

Reply:
Actually, I see that this is a problem in a bunch of places. This is optional!

Reply:
I'll save the qualitative changes for smaller improvement-related PRs :)